RecoTracker/LSTCore/interface/TripletsSoA.h

```diff
@@ -13,10 +13,12 @@ namespace lst {
                       SOA_COLUMN(Params_T3::ArrayU16xLayers, lowerModuleIndices),  // lower module index in each layer
                       SOA_COLUMN(Params_T3::ArrayU8xLayers, logicalLayers),        // layer ID
                       SOA_COLUMN(Params_T3::ArrayUxHits, hitIndices),              // hit indices
+                      SOA_COLUMN(Params_Embedding::ArrayFx6, embedding),
```

Comment:
is this here for a reason? I'm not sure this is filled in this PR

Reply:
Leftover from the t3 embeddings I started with, I'll remove it.